misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -368,9 +367,6 @@ abstract class TransacterTest {
 
   @Test
   fun constraintViolationCausesTransactionToRollback() {
-    // Uniqueness constraints aren't reliably enforced on Vitess
-    assumeTrue(!transacter.config().type.isVitess)
```

Comment:
If this test is run on non-Vitess databases, should it stay and within an if block that stops it from running for Vitess?

Reply:
I'm not sure I follow.  This block was skipping this test when we're running Vitess.  We want to run this test for Vitess as well.

Reply:
Got it, thanks